### ISSUE TITLE
file_get_contents fails to get versions of Drupal in site:new

### ISSUE THREAD

**Issue by fiasco** [hechoendrupal/DrupalConsole#1063]
```bash
$ php -v
PHP 5.5.23 (cli) (built: Mar 23 2015 01:47:37) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2015 Zend Technologies

$ drupal --version
Drupal Console version 0.9.4

$ drupal site:new angular
[+] Getting releases for Drupal
[+] file_get_contents(https://www.drupal.org/node/3060/release?api_version%5B%5D=7234): failed to open stream: operation failed

$ php -a
Interactive shell

php > print substr(file_get_contents('https://www.drupal.org/node/3060/release?api_version%5B%5D=7234'), 0, 100);
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" version="XHTML+RDFa 1.0" di
```

I'm not sure whats going on but it seems the environment that console is running in is not allowing file_get_contents to open a stream.

**Comment by jmolivas** [comment]
@fiasco: Fixed with https://github.com/hechoendrupal/DrupalConsole/pull/1065

**Comment by jmolivas** [comment]
Fixed on v0.9.5 https://github.com/hechoendrupal/DrupalConsole/releases/tag/0.9.5
